test/index.test.js

```diff
@@ -79,14 +79,16 @@ describe('release-drafter', () => {
         it('creates a release draft', async () => {
           getConfigMock('config-non-master-branch.yml')
 
+          nock('https://api.github.com')
+            .post('/graphql', body =>
+              body.query.includes('query findCommitsWithAssociatedPullRequests')
+            )
+            .reply(200, require('./fixtures/graphql-commits-no-prs.json'))
+
           nock('https://api.github.com')
             .get('/repos/toolmantim/release-drafter-test-project/releases')
```

Comment:
Could the releases API and content of `.github/release-drafter.yml` not be replaced by a single GraphQL call?